transport/src/main/java/io/camunda/zeebe/transport/stream/impl/ClientStream.java

```diff
@@ -17,4 +19,22 @@
     AggregatedClientStream<M> serverStream,
     DirectBuffer streamType,
     M metadata,
-    ClientStreamConsumer clientStreamConsumer) {}
+    ClientStreamConsumer clientStreamConsumer) {
+
+  ActorFuture<Void> push(final DirectBuffer payload, final ConcurrencyControl executor) {
+    final ActorFuture<Void> result = executor.createFuture();
```

Comment:
Passing the executor here is a bit wierd. But it is done to help use TestActorFuture in tests.